mlvm/scripts/verify_dask_yarn.py

```diff
@@ -1,11 +1,8 @@
 from dask_yarn import YarnCluster
-from dask.distributed import Client
 import dask.array as da
 
 import numpy as np
 
 cluster = YarnCluster()
-client = Client(cluster)
```

Comment:
please stop.  You can't change the tests unless the API has changed.  The tests need to stay the same or may be changed to use more modern API methods if those have changed.

Reply:
They were not being used in the script, so thought of removing them.